ecosystem/sep-0024.md

```diff
@@ -0,0 +1,688 @@
+    * If an `amount` parameter is provided to the interactive URL, make sure to use that value to pre-populate the amount on the interactive form, since this greatly improves the user experience.
+    * In order to fulfill a withdrawal, a wallet must make a payment to the Stellar address that the anchor provides. It is the anchor's job to watch for Stellar payments to the given address and make the external transaction as soon as they're detected. It is recommended for anchors to listen to at least `payment` and `path_payment` [operations](https://www.stellar.org/developers/horizon/reference/endpoints/payments-all.html), although supporting the remaining ones is important for covering all use cases. Most Stellar SDKs already support listening to all payment forms via streaming.
+    * Some wallets might exchange currencies only once they're ready to send the withdrawal payment, so there might be slight fluctuations of value between the informed withdrawal `amount` and the actual transferred amount. It is recommended for anchors to accept an amount fluctuation of up to ±10%, and  adjust the amount to be transferred (and fees) to reflect the actual value received.
+* **Providing transaction status**
+    * Provide the `/transaction` endpoint. The wallet relies on it to complete interactive withdrawals.
+    * Provide the `/transactions` endpoint. Wallets normally display transaction histories.
+
+## Deposit
+
+A deposit is when a user sends some non-stellar asset (BTC via Bitcoin network, USD via bank transfer, Cash to a teller, etc...) to an account held by an anchor. In turn, the anchor sends an equal amount of tokens on the Stellar network (minus fees) to the user's Stellar account.
+
+The deposit endpoint allows a wallet to get deposit information from an anchor, so a user has all the information needed to initiate a deposit. It also lets the anchor specify additional information that the user must submit interactively via a popup or embedded browser window to be able to deposit.
+
+If the given account does not exist, or if the account doesn't have a trust line for that specific asset, see the [Special Cases](#special-cases) section below.
+
+
+### Request
+
+```
+GET TRANSFER_SERVER/deposit
+```
+
+Request Parameters:
+
+Name | Type | Description
+-----|------|------------
+`asset_code` | string | The code of the asset the user wants to deposit with the anchor. Ex BTC, ETH, USD, INR, etc. This may be different from the asset code that the anchor issues. Ex if a user deposits BTC and receives MyBTC tokens, `asset_code` must be BTC.
```

Comment:
No its because asset_code in this case refers to the off-chain asset. If i want to deposit dollars at anchor usd, i would send asset_code=USD but receive USDX tokens.  This feels broken.

@tomquisel @tomerweller  I think you were talking about this earlier, and the immediate solution for anchors was to just use matching asset codes but i don't think that's viable going forward.  What do you think of scrapping this requirement that the asset_code here is the fiat/off-chain asset code, and just let it be whatever the on-chain code is?  IE i would deposit USDX here with anchorUSD.  We can still link the name of the real-world asset in the stellar.toml under the `anchor_asset` field.
https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0001.md#currency-documentation
`If anchored token, code / symbol for asset that token is anchored to. E.g. USD, BTC, SBUX, Address of real-estate investment property.`

This feels more like what we would expect and shouldn't break any existing implementations, especially if we've guided them all to match the asset_code here to the asset_code they're issuing.